src/App.tsx

```diff
@@ -1,14 +1,50 @@
-/* eslint-disable max-len */
-import React from 'react';
+import { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setLoading] = useState(true);
+  const [filterBy, setFilterBy] = useState('all');
+  const [query, setQuery] = useState('');
+  const [selectedTodoId, setSelectedTodoId] = useState(0);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+
+  useEffect(() => {
+    getTodos()
+      .then(todosFromServer => {
+        setLoading(false);
+        setTodos(todosFromServer);
+      });
```

Comment:
Let’s use async await. It gives you a simpler way to work with asynchronous promise-based code